### DISCUSSION TITLE
Cyclomatic Complexity on Summary?

### DISCUSSION THREAD

**Discussion by the-avid-engineer** [danielpalme/ReportGenerator#533] [Ideas]
It would be awesome if there was an option to have Cyclomatic Complexity (maybe NPath Complexity too?) in the Summary Table and allow the table to be sorted by those columns

This tool is great, by the way. Thanks for making it!

**Comment by danielpalme** [comment]
@the-avid-engineer 
Thanks for your suggestion.

I will consider this for the next release. I guess I will add an option to select custom table columns based on the metrics that are available.

**Comment by danielpalme** [comment]
I took quite I a while, but the feature is now available in version 5.1.14.
Here's a sample report:
https://reportgenerator.io/resources/reports/Html/index.html

You can configure which columns should be visible via the "Select coverage types & metrics" button
